app/main.py

```diff
@@ -1,8 +1,49 @@
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
```suggestion
                 brand: str
             ) -> None:
```

Reply:
please start each parameter from new line starting from self.
opening and closing parenthesis should be on new lines as well

Reply:
fix in all places